client-java/controller/src/main/java/org/evomaster/client/java/controller/AuthUtils.java

```diff
@@ -66,11 +66,11 @@ public static AuthenticationDto getForAuthorizationHeader(String dtoName, String
      * @param password  password for that user
      * @return a DTO
      */
-    public static AuthenticationDto getForDefaultSpringFormLogin(String dtoName, String username, String password){
+    public static AuthenticationDto getForDefaultSpringFormLogin(String dtoName, String username, String password, String endpoint){
```

Comment:
this would break all existing driver that used this function. you can create a new function with same name and parameters, and have old function without `String endpoint` calling this version with `/login` as input (as Java does not have default values for method inputs)

Reply:
do not copy&paste code. the other function with 1 less parameter should remove its code, and call this longer version with `return getForDefaultSpringFormLogin(dtoName, username, password, endpoint,"/login");`